backend/src/main/java/com/crm/exception/GlobalExceptionHandler.java

```diff
@@ -30,12 +30,18 @@ public ErrorResponse handleCustomerNotFoundException(final CustomerNotFoundExcep
         return new ErrorResponse(getSource(request), List.of(e.getMessage()));
     }
 
-    @ResponseStatus(HttpStatus.NOT_FOUND)  
+    @ResponseStatus(HttpStatus.NOT_FOUND)
     @ExceptionHandler(CarNotFoundException.class)
     public ErrorResponse handleCarNotFoundException(final CarNotFoundException e, final HttpServletRequest request) {
         return new ErrorResponse(getSource(request), List.of(e.getMessage()));
     }
 
+    @ResponseStatus(HttpStatus.CONFLICT)
+    @ExceptionHandler(CarException.class)
+    public ErrorResponse handleCarHandlingException(final CarException e, final HttpServletRequest request) {
```

Comment:
i think handleCarException method name is enough

Reply:
Fixed
U're right. :) You can say that again. :)
kur.... więcej czasu straciłem na znalezienie odpowiedniego idiomu. :) .... a i tak nie jest to ten co chciałem pierwotnie. 🥇